### PR TITLE
feat(api): Get all licenses-admin acknowlegments

### LINKED ISSUE #2511: License Admin
Get & Set Acknowledgements & comments (Standard)

### PR DESCRIPTION
**Pull request by dushimsam**
## Description

Added the API to retrieve a list of the licenses' admin acknowledgments.

### Changes

1. Added a new method in  `LicenseController` to build the functionality.
2. Updated  the main file(`index.php`) by adding a new route `GET` `/licenses/adminacknowlegments`.
4. Updated the `openapi.yaml` file  to write the new API's documentation.

## How to test

Make a GET request on the endpoint:  `/licenses/adminacknowlegments`,

## Screenshots

image

### Related Issue:
Fixes #2511 

    
cc: @shaheemazmalmmd @GMishx

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2512"></a>

### CODE REVIEW

File: src/www/ui/api/documentation/openapi.yaml [hunk 1]
```diff
@@ -3704,6 +3729,25 @@ components:
           type: boolean
           description: Add the copyright information as text findings
           default: false
+    AdminLicenseAcknowledgement:
+      type: object
+      properties:
+        la_pk:
```
Comment (GMishx) [inline src/www/ui/api/documentation/openapi.yaml:3735]: Would suggest calling it ID.
```suggestion
        id:
```

**Review by GMishx — CHANGES_REQUESTED** [review]
Minor change.

**Comment by GMishx** [comment]
Tested, working as expected.

File: src/www/ui/api/documentation/openapi.yaml (hunk 1 above)
Comment (dushimsam) [reply src/www/ui/api/documentation/openapi.yaml]: Done

**Review by GMishx — APPROVED** [review]
Changes looks good.

**Pull request merged**